content/docs/getting-started/one-off-tasks.md

```diff
@@ -54,24 +61,19 @@ Note that this will not work for any command that is interactive.
         command: (<your command> && echo SUCCESS || echo FAIL) && sleep infinity
         ```
 
-1. Deploy the one-off app, and view the output:
-
-   When deploying the one-off tasks, it's important to disable health-checks and
+3. Deploy the one-off app, and view the output. When deploying the one-off tasks, it's important to disable health-checks and
    routes in order to prevent deployment issues during the buildpack phase and
    having multiple applications with the same mapped route respectively. For
    more information on these options, see the [`--no-route`][cf-no-route] and
    [`--health-check-type`][cf-health-check] documentation. In order to keep
    changes to your copied manifest at a minimum, you can provide these
-   configuration options directly on the command-line.
-
-    ```sh
-    cf push -f task_manifest.yml --health-check-type none --no-route
-    cf logs --recent task-runner
-    ```
-1. If needed, use [`cf files`][] to collect any artifacts.
-1. Run `cf delete task-runner` to clean it up. **If you don't do this, your short-lived app may automatically run itself again in the future.** cloud.gov sometimes automatically restarts apps as part of routine operations (such as platform updates), which can include restarting this kind of app if it hasn't been deleted.
-
-[`cf files`]: http://cli.cloudfoundry.org/en-US/cf/files.html
+   configuration options directly on the command-line:
+```sh
+cf push -f task_manifest.yml --health-check-type none --no-route
+cf logs --recent task-runner
+```
+4. If needed, use [`cf ssh`]({{< relref "docs/apps/using-ssh.md" >}}) to collect any artifacts.
+5. Run `cf delete task-runner` to clean it up. **If you don't do this, your app may automatically run itself again in the future.** cloud.gov sometimes automatically restarts apps as part of routine operations (such as platform updates), which can include restarting this kind of app if it hasn't been deleted.
```

Comment:
All this looks good, but could we edit this `markdown` to have numbered bullet points as `1., 1., 1., 1.`. Markdown will auto-increment them for us and make diffs better in the future.

Reply:
@rogeruiz Good point, done!